app/views/admin/get_involved/index.html.erb

```diff
@@ -1,9 +1,26 @@
-<% page_title 'Get involved' %>
+<% content_for :page_title, "Get involved" %>
+<% content_for :title, "Get involved" %>
+<% content_for :title_margin_bottom, 2 %>
 
-<div class="get-involved-header">
-  <h1>Get involved</h1>
-  <%= link_to "View on website", get_involved_url %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <p class="govuk-body">
+      <%= link_to "View on website", get_involved_url, class: "govuk-link" %>
+    </p>
+
+    <%= render "components/secondary_navigation", {
+      aria_label: "Get involved tabs",
+      items: [
+        {
+          label: "Get involved",
+          href: admin_get_involved_path,
+          current: true,
```

Comment:
@davidgisbey   Do these need paths to be set like this: 

current: path == admin_get_involved_path($INSTANCE) 

If it is I will update into the guide.

Reply:
So this section is a bit awkward because the `take part` pages live in another folder. Let's leave this as it is now, but the two sections probably need to be combined at some point.